source/opt/instruction.h

```diff
@@ -265,7 +280,7 @@ inline const Operand& Instruction::GetOperand(uint32_t index) const {
   return operands_[index];
 };
 
-inline void Instruction::AddOperand(Operand &&operand) {
+inline void Instruction::AddOperand(Operand&& operand) {
   operands_.push_back(operand);
```

Comment:
This probably was supposed to be
operands_.push_back(std::move(operand));

Reply:
Done.